db/data_migration/20150624075140_populate_featured_policies.rb

```diff
@@ -0,0 +1,41 @@
+
+def policies_to_organisations(policies)
+  organisation_content_ids = policies.inject({}) { |memo, policy|
+    orgs = policy["links"]["organisations"]
+    orgs.each { |org| memo[org["content_id"]] = 1 } if orgs
+    memo
+  }.keys
+
+  Organisation.where(content_id: organisation_content_ids)
+end
+
+def top_3_policy_results(organisation)
+  Whitehall.unified_search_client.unified_search(
+    filter_organisations: [organisation.slug],
+    filter_format: "policy",
+    count: "3",
+    order: "-public_timestamp"
+  ).results
+end
+
+def policy_content_id(policies, policy_link)
+  policy = policies.select do |policy|
+    policy["base_path"] == policy_link
+  end.first
```

Comment:
Chaining do/end is against [the styleguide](https://github.com/alphagov/styleguides/blob/master/ruby.md#syntax), use braces.